File: 2574-left-and-right-sum-differences/2574-left-and-right-sum-differences.java
class Solution {
    public int[] leftRightDifference(int[] nums) {
        int sum=0;
        for(int i=0; i<nums.length; i++){
            sum+=nums[i];
        }
        int lt=0;
        for(int i=0; i<nums.length; i++){
            sum-=nums[i];
            int val=nums[i];
            nums[i]=Math.abs(sum-lt);
            lt+=val;
        }
        return nums;
    }
}

File: 2574-left-and-right-sum-differences/2574-left-and-right-sum-differences.py
class Solution:
    def leftRightDifference(self, nums: List[int]) -> List[int]:
        totSum=sum(nums)
        lt=0
        for i in range(len(nums)):
            val=nums[i]
            totSum-=nums[i]
            nums[i]=abs(totSum-lt)
            lt+=val
        return nums